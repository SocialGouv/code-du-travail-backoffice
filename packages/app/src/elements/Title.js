import styled from "@emotion/styled";

const Title = styled.h1`
  color: black;
  font-size: 1.3rem;
  font-weight: 600;
  margin: 1rem 0;
  user-select: none;
`;

export default Title;
