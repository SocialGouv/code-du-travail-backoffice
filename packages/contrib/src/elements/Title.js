import styled from "styled-components";

export default styled.p`
  color: black;
  font-size: ${props => (Boolean(props.isSecondary) ? "1.1" : "1.25")}rem;
  font-weight: 600;
  margin: ${props => (props.isFirst ? "0 0 0.5rem" : "1.5rem 0 0.5rem")};
  user-select: none;
`;
