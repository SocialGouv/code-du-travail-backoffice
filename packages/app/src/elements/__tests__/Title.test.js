import React from "react";

import Title from "../Title";

describe("elements/<Title />", () => {
  const TEXT = "A Title";

  it(`should pass`, () => {
    const $title = testRender(<Title>{TEXT}</Title>);

    expect($title).toMatchSnapshot();

    expect($title).toHaveStyleRule("margin", "1rem 0");
  });
});
