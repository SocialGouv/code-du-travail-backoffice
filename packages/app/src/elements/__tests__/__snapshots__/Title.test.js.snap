// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/<Title /> should pass 1`] = `
.emotion-0 {
  color: black;
  font-size: 1.3rem;
  font-weight: 600;
  margin: 1rem 0;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

<h1
  className="emotion-0 emotion-1"
>
  A Title
</h1>
`;
